pkg/s3-proxy/config/managerimpl.go

```diff
@@ -20,9 +20,6 @@ import (
 	"github.com/thoas/go-funk"
 )
 
-// Main configuration folder path.
-var mainConfigFolderPath = "conf/"
```

Comment:
This var is used in tests. Removing it without any edit on tests will break them.

See here an example: https://github.com/oxyno-zeta/s3-proxy/blob/master/pkg/s3-proxy/config/managerimpl_integration_test.go#L1524

Can you please edit tests to make them work with your changes ?